src/workspaces/SettingsModal/SettingsModal.tsx

```diff
@@ -196,13 +211,26 @@ const SettingsModal = (props: SettingsModalProps): ReactNode => {
     ) {
       // If the bucket had no soft delete setting before, and the current one is the default retention, don't event.
     } else if (!_.isEqual(originalSoftDeleteSetting, newSoftDeleteSetting)) {
-      // Event if an explicit setting existed before and it changed.
+      // Event if the setting changed.
```

Comment:
(changed the wording here because it also events if no explicit setting existed before and the new setting is non-default)